config/config.go

```diff
@@ -331,8 +342,11 @@ func (c *Cluster) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.Scheme != "http" && c.Scheme != "https" {
 		return fmt.Errorf("`cluster.scheme` must be `http` or `https`, got %q instead for %q", c.Scheme, c.Name)
 	}
-	if c.HeartBeatInterval == 0 {
-		c.HeartBeatInterval = Duration(time.Second * 5)
```

Comment:
Does this mean that default value for heartbeat interval will change if it is unset in the config after the upgrade?

Reply:
This has already been done in `testdata/default_values.yml` section
https://github.com/Vertamedia/chproxy/pull/60/commits/36ba0b4400f59410ba3b9bce98ec22715dc71d4d#diff-bfeae88573cf9afa7638e19a4f7b4535R238